packages/mermaid/src/diagrams/info/infoDetector.ts

```diff
@@ -12,13 +12,13 @@ const detector: DiagramDetector = (txt) => {
 
 const loader: DiagramLoader = async () => {
   const { diagram } = await import('./infoDiagram.js');
-  return { id, diagram };
+  return { id: id, diagram: diagram };
 };
 
 const plugin: ExternalDiagramDefinition = {
-  id,
-  detector,
-  loader,
+  id: id,
+  detector: detector,
+  loader: loader,
 };
```

Comment:
```ts
{ id, diagram }
{ id: id, diagram: diagram }
```

Both are equivalent. The first one is a concise syntax.

Reply:
Use the shorter syntax everywhere (that you changed). Let's leave the rest of instances as-is for now.

Reply:
I'll fix them now.